laplace/curvature/backpack.py

```diff
@@ -85,6 +85,41 @@ def gradients(self, x, y):
                         for p in self._model.parameters()], dim=1)
         return Gs, loss
 
+    def gp_jacobians(self, X):
+        """
+        A helper function to compute jacobians (used in GP methods to reduce code duplication).
+        """
+        if self.last_layer:
+            Js, f = self.last_layer_jacobians(self.model, X)
+        else:
+            Js, f = self.jacobians(self.model, X)
+        return Js, f
```

Comment:
having this helper function enables me to reuse same kernel methods (`kernel_batch`, `kernel_star`, `kernel_batch_star`) in both `FunctionalLaplace` and `FunctionalLLLaplace` and hence avoid code duplication or having same if-else branch in all kernel methods. however, I agree that it's better to move it to `FunctionalLaplace` instead of having it in `BackPackInterface`, will change it